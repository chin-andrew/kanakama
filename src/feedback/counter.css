.counter {
  justify-content: center;
  display: flex;
}

.counter__column {
  margin: 16px;
  display: flex;
  flex-direction: column;
  align-items: center;
  font-size: 20px;
}

.counter__score {
  font-size: 28px;
  font-weight: bold;
}