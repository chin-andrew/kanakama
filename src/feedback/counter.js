import React from 'react';
import './counter.css';

export default (correct, incorrect) => {
  return (
    <div className='counter'>
      <div className='counter__column'>
        <div>
          Correct:
        </div>
        <div className='counter__score'>
          {correct}
        </div>
      </div>
      <div className='counter__column'>
        <div>
          Incorrect:
        </div>
        <div className='counter__score'>
          {incorrect}
        </div>
      </div>
    </div>
  )
}