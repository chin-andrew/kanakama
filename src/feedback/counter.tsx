import React from 'react';
import './counter.css';

export default (correct: number, incorrect: number) => {
  return (
    <div className='counter'>
      <div className='counter__column'>
        <div>
          Correct:
        </div>
        <div className='counter__score'>
          {correct}
        </div>
      </div>
      <div className='counter__column'>
        <div>
          Incorrect:
        </div>
        <div className='counter__score'>
          {incorrect}
        </div>
      </div>
    </div>
  )
}