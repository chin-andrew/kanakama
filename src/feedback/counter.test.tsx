import renderer from 'react-test-renderer';
import counter from './counter';

describe('counter tests', () => {
  it('renders a counter', () => {
    const tree = renderer.create(counter(0, 0)).toJSON();
    expect(tree).toMatchSnapshot();
  })
})