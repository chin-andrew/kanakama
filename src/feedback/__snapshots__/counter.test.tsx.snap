// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counter tests renders a counter 1`] = `
<div
  className="sc-bdVaJa NFzwl"
  id="counter"
>
  <div
    className="sc-bwzfXH giRpJe"
  >
    <span
      className="sc-htpNat fIjutD"
    >
      Correct:
    </span>
    <span
      className="sc-bxivhb dzCqNz"
    >
      0
    </span>
  </div>
  <div
    className="sc-bwzfXH giRpJe"
  >
    <span
      className="sc-htpNat fIjutD"
    >
      Incorrect:
    </span>
    <span
      className="sc-bxivhb dzCqNz"
    >
      0
    </span>
  </div>
</div>
`;
